### PR TITLE
Optimize APK size

### PR DESCRIPTION
**Pull request by fabiogrum**
Optimize APK size by using isShrinkResources, resourceConfigurations and removing unused libraries.

Screenshot 2023-07-31 at 22 46 31

### CODE REVIEW

**Comment by MrAdkhambek** [comment]
@fabiogrum I think we can remove Kotlin also

**Comment by MrAdkhambek** [comment]
image

File: app/build.gradle.kts [hunk 1]
```diff
@@ -15,12 +15,15 @@ android {
         versionCode = 1
         versionName = "1.0"
 
+        resourceConfigurations += setOf("en")
```
Comment (ShikaSD) [inline app/build.gradle.kts:17]: nice! :)

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,24 +1,13 @@
...
+activity-ktx = "1.7.2"
 
 [libraries]
-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "core-ktx" }
-junit = { group = "junit", name = "junit", version.ref = "junit" }
-androidx-test-ext-junit = { group = "androidx.test.ext", name = "junit", version.ref = "androidx-test-ext-junit" }
-espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "espresso-core" }
-appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "appcompat" }
-material = { group = "com.google.android.material", name = "material", version.ref = "material" }
+activity-ktx = { group = "androidx.activity", name = "activity-ktx", version.ref = "activity-ktx" }
```
Comment (ShikaSD) [inline gradle/libs.versions.toml]: tbh, we don't really need "androidx.activity" either :)

**Review by ShikaSD — COMMENTED** [review]
Nice work!
Could you remove androidx.activity as well? Just extending `android.app.Activity` should be sufficient, I believe

File: gradle/libs.versions.toml (hunk 2 above)
Comment (fabiogrum) [reply gradle/libs.versions.toml]: Done ;)

**Comment by fabiogrum** [comment]
> @fabiogrum I think we can remove Kotlin also

@MrAdkhambek It was a while since I wrote any Java!

**Pull request merged**